Misc/NEWS.d/next/Core and Builtins/2017-08-10-17-32-48.bpo-31179.XcgLYI.rst

```diff
@@ -0,0 +1 @@
+Make dict.copy() up to 5.5 times faster.
```

Comment:
The one case it slowed down was an empty dict ... is it worth a special path to drop out and just create a new empty dict then?

Reply:
Added a fast-path for empty dicts.